OrbitGrpcProtos/capture.proto

```diff
@@ -104,7 +104,7 @@ message InternedTracepointInfo {
 message TracepointEvent {
   int32 pid = 1;
   int32 tid = 2;
-  int64 time = 3;
+  uint64 time = 3;
```

Comment:
Why this change? Seems to be unrelated to the rest?

Reply:
Hah, good point. I think that is a left over from extracting the change from my branch. I'd actually like to keep it in, as this might clearly be an issue, though.

Reply:
I would not leave changes in a PR with a different purpose, in particular if they have the potential to break things.

Reply:
Okay, agree.

Reply:
Removed it.